gpu-operator/google-gke.rst

```diff
@@ -67,11 +86,112 @@ Prerequisites
   in the Google Cloud documentation.
 
 
-*********
-Procedure
-*********
+*********************************
+Using the Google Driver Installer
+*********************************
+
+Perform the following steps to create a GKE cluster with the ``gcloud`` CLI and use Google driver installer to manage the GPU driver.
+You can create a node pool that uses a Container-Optimized OS node image or a Ubuntu node image.
+
+#. Create the cluster.
+   Refer to `Running GPUs in GKE Standard clusters <https://cloud.google.com/kubernetes-engine/docs/how-to/gpus#create>`__
+   in the GKE documentation.
+
+   When you create the cluster, specify the following additional ``gcloud`` command-line options:
+
+   - ``--node-labels="gke-no-default-nvidia-gpu-device-plugin=true"``
+
+     The node label disables the GKE GPU device plugin daemon set on GPU nodes.
+
+   - ``--accelerator type=...,gpu-driver-version=disabled``
+
+     This argument prevents automatically installing the GPU driver on GPU nodes.
+
+#. Get the authentication credentials for the cluster:
+
+   .. code-block:: console
+
+      $ USE_GKE_GCLOUD_AUTH_PLUGIN=True \
+          gcloud container clusters get-credentials demo-cluster --zone us-west1-a
```

Comment:
small nit, we recommend users to avoid zonal clusters, and instead use regional clusters in general.

So I would perhaps put `--location=us-west1` instead.